.github/workflows/build.yaml

```diff
@@ -0,0 +1,63 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    check:
+        runs-on: ubuntu-20.04
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Composer check
+              run: composer validate --strict --no-check-lock
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - description: 'No Symfony specified'
+                      php: '7.2'
+                    - description: 'No Symfony specified'
+                      php: '7.3'
+                    - description: 'No Symfony specified'
+                      php: '7.4'
+                    - description: 'No Symfony specified'
+                      php: '8.0'
+                    - description: 'Lowest deps'
+                      php: '7.2'
+                      composer_option: '--prefer-lowest'
+                      env:
+                          SYMFONY_DEPRECATIONS_HELPER: max[self]=0
+                    - description: 'Symfony 4.4'
+                      php: '7.3'
+                      symfony: 4.4.*
+                    - description: 'Symfony 5.2'
```

Comment:
I suggest removing this job to keep only explicit jobs for LTS versions (thus, 5.2 is currently the latest Symfony version and so already used in normal jobs).

Reply:
On the other hand, a job testing the latest 3.4 LTS might make sense as it is still supported in composer.json.